lib/rubocop/sorbet/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module RuboCop
   module Sorbet
-    VERSION = "0.10.5"
+    VERSION = File.read(File.expand_path("../../../VERSION", __dir__)).strip
```

Comment:
We should not reading reading a file when loading this file just to get a string. Why do you need this?

Reply:
2 things need the version: the gem's `VERSION` constant and the `prepare_release` rake task. So we have both this file and the `VERSION` file as the version source.
And since we're adding a check on version change, I think it's better to have just one source of truth.
Comparing with parsing `version.rb`'s `VERSION` constant, IMO it's easier to directly read from the `VERSION` file, so I decided to use it as the single version source.